<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1482</width>
    <height>899</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="experimentNameLabel">
               <property name="text">
                <string>Experiment Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="experimentNameLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="dateLabel">
               <property name="text">
                <string>Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="dateTimeEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="MeasurePeriodLabel">
               <property name="text">
                <string>Measurement Period [s]: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="MeasureFreqLineEdit">
               <property name="placeholderText">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="startPushButton">
               <property name="text">
                <string>Start</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pausePushButton">
               <property name="text">
                <string>Pause</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="endAndSavePushButton">
               <property name="text">
                <string>End / Save</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QFrame" name="frame_8">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="opaqueResize">
              <bool>true</bool>
             </property>
             <widget class="QMdiArea" name="mdiArea">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
             <widget class="QMdiArea" name="mdiArea_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>1</horstretch>
                <verstretch>1</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_7">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Engineer's Notes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="errorDisplay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1482</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionSerial_Instrument"/>
     <addaction name="actionGPIB_Instrument"/>
     <addaction name="actionEthernet_Instrument"/>
     <addaction name="actionUSB_Instrument"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionView"/>
    <addaction name="actionDeviceList"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="action_Add_Plot_Window"/>
    <addaction name="action_Open_Old_Plot"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionExperiment"/>
    <addaction name="action_gasValve"/>
    <addaction name="action_pressureGauge"/>
    <addaction name="action_lockInAmplifier1"/>
    <addaction name="action_lockInAmplifier2"/>
    <addaction name="action_temperatureController"/>
    <addaction name="action_magnetPowerSupply"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Add_Plot_Window">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_open_old_plots">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Open_Old_Plot">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExperiment">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Experiment</string>
   </property>
  </action>
  <action name="action_gasValve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gas Valve</string>
   </property>
  </action>
  <action name="action_pressureGauge">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pressure Gauge</string>
   </property>
  </action>
  <action name="action_lockInAmplifier1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock-In Amplifier</string>
   </property>
  </action>
  <action name="action_temperatureController">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Temperature Controller</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionDeviceList">
   <property name="text">
    <string>Device List</string>
   </property>
  </action>
  <action name="actionSerial_Instrument">
   <property name="text">
    <string>Serial Instrument</string>
   </property>
  </action>
  <action name="actionGPIB_Instrument">
   <property name="text">
    <string>GPIB Instrument</string>
   </property>
  </action>
  <action name="actionUSB_Instrument">
   <property name="text">
    <string>USB-6525 Instrument</string>
   </property>
  </action>
  <action name="actionEthernet_Instrument">
   <property name="text">
    <string>Ethernet Instrument</string>
   </property>
  </action>
  <action name="action_magnetPowerSupply">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Magnet Power Supply</string>
   </property>
  </action>
  <action name="action_lockInAmplifier2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock-In Amplifier 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
